packages/wallets/src/metamask/metamask.page.ts

```diff
@@ -394,7 +394,12 @@ export class MetamaskPage implements WalletPage {
   async changeWalletAddress(addressName: string) {
     await this.navigate();
     await this.page.click('data-testid=account-menu-icon');
-    await this.page.click(`text=${addressName}`);
+    await this.page.click(`section:has-text("${addressName}")`);
+    const accountNumber = this.page.locator(
+      `button[data-testid="account-menu-icon"]:has-text("${addressName}")`,
+    );
+    await accountNumber.waitFor({ state: 'visible', timeout: 2000 });
+    await this.page.waitForTimeout(2000);
```

Comment:
I've thought, maybe we can change `ConnectWallet()` instead of using that function. 
I mean [that function](https://github.com/lidofinance/wallets-testing-modules/blob/bbe1869da3f1e03b90849d3890a609af9bf8fe79/packages/wallets/src/metamask/metamask.page.ts#L269).
In that function, we can choose the first account by default, and maybe then we can abandon this `changeWallet Address`

Reply:
I gues we can leave it on case if would have issues with this since approach to use it once(and looks like for now it works) better than extend function to use it several times